node/api/server.go

```diff
@@ -39,7 +39,7 @@ type PodHandlerConfig struct {
 }
 
 // PodHandler creates an http handler for interacting with pods/containers.
-func PodHandler(p PodHandlerConfig, debug bool) http.Handler {
+func PodHandler(p PodHandlerConfig, debug bool, cfg ...StreamTimeOutConfig) http.Handler {
```

Comment:
Sorry I was thinking something like functional options so `...func(*Config)` And then you can provide a list of functions to set the config... e.g. `WithStreamIdleTimeout(dur)`.

But I think we could probably just add a stream config to the `PodHandlerConfig`

Reply:
Updated, thanks